### PR TITLE
Create CIC & Command & Flight Crew Vendors

### PR DESCRIPTION
**Pull request by DangerRevolution**
## About the PR

This adds the Executive Officer Equipment and Weapon Vendor; Staff Officer Equipment and Weapon Vendor as well as the Commanding Officer weapon & vendor and upon request all the flight Crew Vendors

They're still lacking entries; hence I won't close the issues related to these vendors but they are mostly complete. Depends how long this takes to review; I've left `TODO RMC14` on items that need creating such as laser designators, and left my own TODOs for clothes I need to port in a separate PR. 

The CO weapons have been replaced with generic equivalents; just so the category isn't empty. I can comment out the entires if desired.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

^ I can't show the CO vend as I have no idea how to whitelist myself; but I will screenshot the other Vends.

image
image
image
image
image
image

**Changelog**

:cl: DangerRevolution
- add: Added the Commanding Officer's equipment and weapon vendors.
- add: Added the Executive Officer's equipment and weapon vendors.
- add: Added the Staff Officer's equipment and weapon vendors
- add: Added the Pilot & Dropship Crew Chiefs' equipment and weapon vendors.

### CODE REVIEW

**Comment by DangerRevolution** [comment]
setting this to draft as i've discovered where the CO clothing vend is

**Comment by DangerRevolution** [comment]
reopened because i added clothing vend

File: Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/command.yml [hunk 1]
```diff
@@ -0,0 +1,599 @@
...
+        points: 5
+      - id: RMCFlashlight
+        points: 1
+
+
+# Commanding Officer #
+
+# Clothing Vend
+
+- type:
```
Comment (DangerRevolution) [inline Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/command.yml:373]:
```suggestion
- type: entity
```

File: Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/command.yml [hunk 2]
```diff
@@ -0,0 +1,599 @@
...
+# Weapon Vend
+- type: entity
+  parent: ColMarTechBase
+  id: ColMarTechCommandingOfficerWeapon
+  name: ColMarTech Commanding Officer Weapon Rack
+  description: An automated weapons rack for the Commanding Officer. It features a robust selection of weaponry meant only for the UNMC's top officers.
+  components:
+  - type: AccessReader
+    access:
+    - [ "CMAccessCommand" ]
```
Comment (DangerRevolution) [inline Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/command.yml:509]:
```suggestion
    - [ "ShipMasterAccess" ]
```

**Comment by DangerRevolution** [comment]
PR should be fixed and ready for le review

File: Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/auxiliary.yml [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+# Equipment Rack
+- type: entity
+  parent: ColMarTechBase
+  id: ColMarTechDropshipCrewEquipment
+  name: ColMarTech Dropship Crew Equipment Rack
+  description: An automated equipment vendor for Dropship Crew
+  components:
+  - type: AccessReader
+    access:
+    - [ "CMAccessDropship" ]
```
Comment (DrSmugleaf) [inline Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/auxiliary.yml:12]: This needs to be pilot, not dropship, dropship is the ability to open the doors in general which all marines have
```suggestion
    - [ "CMAccessPilot" ]
```

**Review by DrSmugleaf — APPROVED** [review]
Thanks for the PR!

**Pull request merged**